packages/ui/src/message-pane-input/components/ToolbarBottom.jsx

```diff
@@ -97,17 +97,20 @@ const ToolbarBottom = props => {
     }
   };
   function handleSubmit() {
-    const url = 'http://localhost:9000/uploadFile';
+    alert("Uploading File(s)")
+    const url = process.env.url;
     const formData = new FormData();
-    formData.append('file', file);
-    formData.append('fileName', file.name);
+    for (let i = 0; i < files.length; i++) {
+      formData.append(files[i].name, files[i])
+    }
+    
     const config = {
       headers: {
         'content-type': 'multipart/form-data',
       },
     };
     axios.post(url, formData, config).then((response) => {
-      console.log(response.data);
+      alert("File(s) has been uploaded")
```

Comment:
Use a toast notification for this.

Reply:
Also, you can use a loader to show the progress of the upload if you can't do the progress bar, but ensure you check the package.json to see if a toast notification library has not been installed before installing one.

Reply:
Alright